react-native-cli/index.js

```diff
@@ -232,7 +232,7 @@ function runVerbose(root, projectName, rnPackage) {
   var proc = spawn('npm', ['install', '--verbose', '--save', getInstallPackage(rnPackage)], {stdio: 'inherit'});
```

Comment:
{stdio: 'inherit'} is more useful because it is printing log data while running npm install.
Unlike the `run` command above